static/HtmlFormat.css

```diff
@@ -29,27 +29,39 @@ html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, blockq
 /* Section */
  section {
 	background-color: #FFFFFF;
-	width: 60%;
+	width: 95%;
 	margin: 30px auto;
 	padding: 15px 15px;
 	text-align: center;
 	box-shadow: 0 6px 15px rgba(0, 0, 0, 0.09);
 	border-radius: 4px;
 }
  section > h2 {
-	font-size: 200%;
+	font-size: 125%;
 	font-weight: bold;
 	text-align: center;
+	word-break: break-all;
 }
  h1.pagetitle {
 	margin: 40px 0 20px;
-	font-size: 300%;
+	font-size: 150%;
 	font-weight: bold;
 	text-align: center;
 	color: #2196F3;
 }
  h1.pagetitle > a {
 	color: #2196F3;
+}
+ @media screen and (min-width: 767px) {
```

Comment:
As mentioned before, please place all style changes related to mobile devices in this selector and keep everything outside the selector unchanged. Right now you have changed both styles.